platform/viewer/src/appExtensions/MeasurementsPanel/index.js

```diff
@@ -10,7 +10,7 @@ export default {
   preRegistration({ servicesManager, configuration = {} }) {
     init({ servicesManager, configuration });
   },
-  getPanelModule() {
+  getPanelModule({ servicesManager }) {
```

Comment:
Should we pass this in as `servicesManager` or `ServicesManager`?

Reply:
I'm using camel here because it's an instance of ServicesManager